bin/brain-calc.js

```diff
@@ -1,386 +1,51 @@
-            }
-          }
-          if (randomItemThree === '*') {
-            if ((randomNumberFive * randomNumberSix) === Number(numberUser)) {
-              console.log('Correct!');
-              console.log(`Congratulations, ${nameRequest}!`);
-            } else {
-              console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberFive * randomNumberSix}'`);
-              console.log(`Let's try again, ${nameRequest}!`);
-            }
-          }
-        } else {
-          console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberThree - randomNumberFour}'`);
-          console.log(`Let's try again, ${nameRequest}!`);
-        }
-      }
-      if (randomItemTwo === '*') {
-        if ((randomNumberThree * randomNumberFour) === Number(numberUser)) {
-          console.log('Correct!');
-          console.log(`Question: ${randomNumberFive} ${randomItemThree} ${randomNumberSix}`);
-          const numberUser = readlineSync.question('Your answer: ');
-          if (randomItemThree === '+') {
-            if ((randomNumberFive + randomNumberSix) === Number(numberUser)) {
-              console.log('Correct!');
-              console.log(`Congratulations, ${nameRequest}!`);
-            } else {
-              console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberFive + randomNumberSix}'`);
-              console.log(`Let's try again, ${nameRequest}!`);
-            }
-          }
-          if (randomItemThree === '-') {
-            if ((randomNumberFive - randomNumberSix) === Number(numberUser)) {
-              console.log('Correct!');
-              console.log(`Congratulations, ${nameRequest}!`);
-            } else {
-              console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberFive - randomNumberSix}'`);
-              console.log(`Let's try again, ${nameRequest}!`);
-            }
-          }
-          if (randomItemThree === '*') {
-            if ((randomNumberFive * randomNumberSix) === Number(numberUser)) {
-              console.log('Correct!');
-              console.log(`Congratulations, ${nameRequest}!`);
-            } else {
-              console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberFive * randomNumberSix}'`);
-              console.log(`Let's try again, ${nameRequest}!`);
-            }
-          }
-        } else {
-          console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberThree * randomNumberFour}'`);
-          console.log(`Let's try again, ${nameRequest}!`);
-        }
-      }
-    } else {
-      console.log(`'${numberUser}' is wrong answer ;(. Correct answer was '${randomNumberOne - randomNumberTwo}'`);
-      console.log(`Let's try again, ${nameRequest}!`);
-    }
+const askQuestion = (name) => {
+  const x = genInt();
+  const y = genInt();
+  const sign = signs[Math.floor(Math.random() * signs.length)];
```

Comment:
```suggestion
const askQuestion = (name) => {
  const x = genInt();
  const y = genInt();
  const z = genInt();
  const sign = signs[Math.floor(Math.random() * signs.length)];
```

А не лучше сделать вот так?